modules/localization/msf/common/util/file_utility.cc

```diff
@@ -0,0 +1,120 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#include "modules/localization/msf/common/util/file_utility.h"
+#include <dirent.h>
+#include <errno.h>
+#include <limits.h>
+#define BOOST_NO_CXX11_SCOPED_ENUMS
+#include <boost/filesystem.hpp>
+#undef BOOST_NO_CXX11_SCOPED_ENUMS
+#include <algorithm>
+#include <iostream>
+#include "fastrtps/TopicDataType.h"
+
+namespace apollo {
+namespace localization {
+namespace msf {
+const size_t BUFFER_SIZE = 20480000;
```

Comment:
use kBufferSize type of naming convention

Reply:
fixed